packages/amplication-server/src/core/git/git.provider.service.ts

```diff
@@ -329,7 +335,8 @@ export class GitProviderService {
 
   // installation id flow (GitHub ONLY!)
   async createGitOrganization(
-    args: CreateGitOrganizationArgs
+    args: CreateGitOrganizationArgs,
+    currentUser: User
```

Comment:
There is no need to pass the whole user, only account id is actually used. Also, it won't be necessary to import User type to the service.

Reply:
no gain in passing just the ID - I am always in favor of sending the entire object, instead of adding parameters one by one per use case (and saving the extra import is not a goal)